evm/auxiliary/user_account_contract/transaction.py

```diff
@@ -163,7 +165,7 @@ def as_signed_transaction(self, private_key):
             nonce=self.nonce,
             min_block=self.min_block,
             max_block=self.max_block,
-            gas_price=self.gas_price,
+            gas_price=self.int_gas_price,
```

Comment:
What does the `int_` prefix stand for?  integer?  internal?

Reply:
I think it stands for internal before `gas_price` is removed from transaction format. Will update it to `gas_price`.